### PR TITLE
[DO-6738] Rm hardcoded config for host

### PR DESCRIPTION
**Pull request by pimpin**
[DO-6738]

[DO-6738]: https://doctolib.atlassian.net/browse/DO-6738?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +22,7 @@ To generate config you can use `rails generate couchbase_orm:config`:
 It will generate this `config/couchbase.yml` for you:
 
     common: &common
-      hosts: localhost
+      connection_string: couchbase://localhost
```
Comment (CedricCouton) [inline README.md:20]: why not hosts?

File: lib/couchbase-orm/railtie.rb [hunk 2]
```diff
@@ -46,11 +46,10 @@ def self.rescue_responses
...
             initializer 'couchbase.setup_connection' do
-                config_file = Rails.root.join('config', 'couchbase.yml')
-                if config_file.file? &&
-                    config = YAML.load(ERB.new(File.read(config_file)).result)[Rails.env]
-                    ::CouchbaseOrm::Connection.options = config.deep_symbolize_keys
-                end
+                ENV['COUCHBASE_BUCKET'] ||= Rails.application.config_for(:couchbase).bucket
+                ENV['COUCHBASE_CONNECTION_STRING'] ||= Rails.application.config_for(:couchbase).connection_string
+                ENV['COUCHBASE_USER'] ||= Rails.application.config_for(:couchbase).username
+                ENV['COUCHBASE_PASSWORD'] ||= Rails.application.config_for(:couchbase).password
```
Comment (CedricCouton) [inline lib/couchbase-orm/railtie.rb]: Is it more in ruby best practice?

Comment (pimpin) [reply lib/couchbase-orm/railtie.rb]: I'm not really sure of this.
It could be great to have the review of old rubyists

Comment (simkim) [reply lib/couchbase-orm/railtie.rb]: I think I would expect the inverse, set rails config from env then use config to initialize

Comment (simkim) [reply lib/couchbase-orm/railtie.rb]: I don't think code should change env variables

File: README.md (hunk 1 above)
Comment (pimpin) [reply README.md:20]: It's the way of configuring the couchbase-ruby-client gem : https://github.com/couchbase/couchbase-ruby-client/blob/main/lib/rails/generators/couchbase/config/templates/couchbase.yml

File: lib/couchbase-orm/railtie.rb (hunk 2 above)
Comment (pimpin) [reply lib/couchbase-orm/railtie.rb]: ok let's me try a better implem.

**Pull request closed without merging**